### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by dorkerdev** [sqlitebrowser/sqlitebrowser#3021] [General]
I have a good number of requests to improve some of the usability issues with this application. However, development doesn't seem very active, perhaps dead outright. Lots of open issues and hardly any check-in activity. It will take some time to compile my issue list, but want to know how far behind the dev team is before I make an attempt.

**Comment by chrisjlocke** [comment]
The 'dev team' is anyone who wants to raise a PR and submit it - it's open source.  
'Real life' has gotten in the way of sitting at a keyboard and developing for a number of people, so I would put development as 'stalled' rather than dead outright.  There was a commit in February, after all.  

While requests are useful, to be honest, they're just going to sit on the pile with the others, and unless they're critical 'this project is unusable unless they're fixed', it's going to take a while to get to them ... the existing 500 have to be tackled first.

**Comment by justinclift** [comment]
Definitely not dead.  I'm personally on leave for about 3.5 more weeks, and will be coming back to this project after that. :smile:
